build-system/tasks/extension-helpers.js

```diff
@@ -597,7 +599,27 @@ function buildExtensionCss(extDir, name, version, options) {
       })
     );
   }
-  promises.push(mainCssBinary);
+  promises.push(mainCssBinaryPromise);
+
+  // Currently JSON.stringifying to allow arrays and strings:
+  // {"wrapper": "bento"} and {"wrapper": ["bento"]}
+  // TODO(alanorozco): We might need a `bento` flag instead.
+  if (options.wrapper && JSON.stringify(options.wrapper).includes('"bento"')) {
+    // TODO(alanorozco): Use a TransformCache
+    const promise = mainCssPromise.then(async (css) => {
+      const startTime = Date.now();
+
+      const bentoName = name.replace(/^amp-/, 'bento-');
+      const renamedCss = await renameSelectorsToBentoTagNames(
+        css,
+        mainCssFilepath
+      );
+      await fs.writeFile(`dist/v0/${bentoName}.css`, renamedCss);
```

Comment:
Other files output from this function go into `build/`, and later moved into `dist/`. 

I skipped `build/` altogether since the copying function expects an `amp-*` pattern. Happy to write into `build/` for consistency if desired, though we'd gain nothing more than consistency if I do so.